### DISCUSSION TITLE
firmware update question

### DISCUSSION THREAD

**Discussion by orangoo** [Jyers/Marlin#1104] [Q&A] [answered]
Hi 

If I want to update the firmware (assuming the current firmware is running perfectly with all the settings chosen and adjusted), do all I need is put the settings back in afterwards.

Or do I have to start again as if I'm installing the firmware for the first time.

thanks

**Comment by Cal-LF** [comment, marked as answer]
Flash updated firmware, reset eprom to defaults. Adjust settings as needed and save to eprom each time changes are made.
